src/olympia/accounts/tests/test_views.py

```diff
@@ -1135,6 +1135,13 @@ def test_display_name_validation(self):
             'display_name': [
                 'Ensure this field has no more than 50 characters.']}
 
+        response = self.patch(
+            data={'display_name': u'\x7F\u20DF'})
+        assert response.status_code == 400
+        assert json.loads(response.content) == {
+            'display_name': [
+                'Must contain at least one printable character.']}
```

Comment:
Let's add just one more test that simply adds one printable character and then succeeds to show that it's indeed working as expected with mixed strings.